### DISCUSSION TITLE
2011/03/15/%E8%B1%86%E8%B1%86%E5%8F%91%E5%87%BA%E7%88%B8%E7%9A%84%E5%A3%B0%E9%9F%B3%E4%BA%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#318] [Announcements]
# 2011/03/15/%E8%B1%86%E8%B1%86%E5%8F%91%E5%87%BA%E7%88%B8%E7%9A%84%E5%A3%B0%E9%9F%B3%E4%BA%86

经过一个月的不懈努力，我终于教会了豆豆发“爸”这个音。最近，老婆天天冲着豆豆叫“妈妈”，我则冲着豆豆叫“爸爸”，我们竞赛看谁先教会豆豆。不过，教豆豆说话从来都是我的工作，不用猜也知道老婆赢不了。

https://blog.qizhen.xyz/2011/03/15/%E8%B1%86%E8%B1%86%E5%8F%91%E5%87%BA%E7%88%B8%E7%9A%84%E5%A3%B0%E9%9F%B3%E4%BA%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

陈小刚
2011年03月30日 16:44 编辑
阮老师的豆豆跟我们家的丁丁当当差不多时候出生的啊，不过我们家的早产，我努力了很久现在只能发出一点奇怪的声音:(

回复
ruanqizhen
2011年03月31日 09:19 编辑
双胞胎啊？

回复
陈小刚
2011年03月31日 16:45 编辑
呵呵，是的，还是个龙凤胎~这辈子也算中个500w了

回复
雨中花
2011年03月20日 11:19 编辑
阮老师，你好，本来来这里是来看你的Labview编程实例的（买了你新出的书），没想到却看到了你记录的宝宝成长日记。呵呵，很意外。我也刚有个宝宝，10.30生日，比你们大一个多月。没有看该看的资料，却把你家宝宝的记录给看完了。忍不住想留言，希望能有机会交流关于Labview和宝宝的话题。

回复
华兹华斯
2011年03月16日 10:09 编辑
感谢阮老师，从您的博客上学到了很多的东西，祝豆豆健康成长，祝合家幸福、欢乐^___,^

回复
